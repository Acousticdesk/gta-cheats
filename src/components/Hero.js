import React from 'react'

export default () => (
  <div className="hero">
    <div className="hero-body">
      <h2 className="title is-1">Cheaterino ğŸ”¥</h2>
      <h3 className="subtitle">Your San Andreas cheats in a neat place.</h3>
    </div>
  </div>
)
