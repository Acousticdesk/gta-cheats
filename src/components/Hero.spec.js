import React from 'react'
import renderer from 'react-test-renderer'
import Hero from './Hero'

describe('Hero Component', () => {
  it('', () => {
    const component = renderer.create(<Hero />)
    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
