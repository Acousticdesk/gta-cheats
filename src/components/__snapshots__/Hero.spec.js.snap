// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Component  1`] = `
<div
  className="hero"
>
  <div
    className="hero-body"
  >
    <h2
      className="title is-1"
    >
      Cheaterino
      <span
        aria-label="fire"
        role="img"
      >
        ðŸ”¥
      </span>
    </h2>
    <h3
      className="subtitle"
    >
      Your San Andreas cheats in a neat place.
    </h3>
  </div>
</div>
`;
